server/consumer.go

```diff
@@ -3375,6 +3378,18 @@ func (o *consumer) loopAndGatherMsgs(qch chan struct{}) {
 
 		// Do actual delivery.
 		o.deliverMsg(dsubj, ackReply, pmsg, dc, rp)
+		if !strings.Contains(string(dsubj), "$memphis") {
```

Comment:
Contains =< HasPrefix it is a more performant function

and I want that this logic will happen only on the cloud, wrapt it all in a function